Packs/Illumio/Integrations/IllumioCore/IllumioCore.py

```diff
@@ -811,6 +914,124 @@ def ip_list_get_command(client: PolicyComputeEngine, args: Dict[str, Any]) -> Co
+    readable_output = prepare_ip_lists_get_output(ip_lists_json)
+
+    return CommandResults(
+        outputs_prefix="Illumio.IPLists",
+        readable_output=readable_output,
+        outputs_key_field="href",
+        raw_response=ip_lists_json,
+        outputs=[remove_empty_elements(ip_list) for ip_list in ip_lists_json],
+    )
+
+
+def ruleset_create_command(client: PolicyComputeEngine, args: Dict[str, Any]) -> CommandResults:
+    """Create a ruleset with unique name.
+
+    Args:
+        client: PolicyComputeEngine to use.
+        args: arguments obtained from demisto.args()
+
+    Returns:
+        CommandResult object
+    """
+    name = args.get("name")
+    validate_required_parameters(name=name)
+
+    response = client.rule_sets.create(body={"name": name, "scopes": [[]]})
+    json_response = response.to_json()
+
+    readable_output = prepare_ruleset_create_output(json_response, name)
+    outputs_response = remove_empty_elements(json_response)
+
+    return CommandResults(
+        outputs_prefix="Illumio.Ruleset",
+        outputs_key_field="href",
+        outputs=outputs_response,
+        readable_output=readable_output,
+        raw_response=json_response
+    )
+
+
+def rule_create_command(client: PolicyComputeEngine, args: Dict[str, Any]) -> CommandResults:
+    """Create and assign rules to a particular ruleset.
+
+    Args:
+        client: PolicyComputeEngine to use.
+        args: Arguments obtained from demisto.args()
+
+    Returns:
+        CommandResults: CommandResults object
+    """
+    ruleset_href = args.get("ruleset_href")
+    providers = args.get("providers")
+    consumers = args.get("consumers")
+    ingress_services = args.get("ingress_services")
+    resolve_providers_as = args.get("resolve_providers_as", "workloads")
+    resolve_consumers_as = args.get("resolve_consumers_as", "workloads")
+
+    validate_required_parameters(ruleset_href=ruleset_href, providers=providers, consumers=consumers)
+    providers = argToList(providers)
+    consumers = argToList(consumers)
+    ingress_services = argToList(ingress_services)
+    resolve_providers_as = argToList(resolve_providers_as)
+    resolve_consumers_as = argToList(resolve_consumers_as)
+
+    # Building rule
+    rule = Rule.build(
+        ingress_services=ingress_services,
+        consumers=consumers, providers=providers,
+        resolve_consumers_as=resolve_consumers_as,
+        resolve_providers_as=resolve_providers_as
+    )
+
+    response = client.rules.create(rule, parent=ruleset_href)
+    response = response.to_json()
+
+    readable_output = prepare_rule_create_output(response)  # type: ignore
+
+    return CommandResults(
+        outputs_prefix="Illumio.Rule",
+        outputs_key_field="href",
+        outputs=remove_empty_elements(response),
+        readable_output=readable_output,
+        raw_response=response
+    )
+
+
 def main():
     """Parse params and runs command functions."""
     try:
```

Comment:
The debug line below is using an f-string - maybe just remove the debug line entirely.

Reply:
Yes, will remove it.